Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
Same

Reply:
Cargo needs a `Cargo.toml` file to build the project.

Reply:
Yes, but it's part of your upstream sources.